gamemodes/terrortown/gamemode/server/sv_gamemsg.lua

```diff
@@ -114,121 +120,132 @@ function GetPlayerFilter(pred)
-	return GetTeamFilter(TEAM_INNOCENT, alive_only)
+function GetInnocentFilter(aliveOnly)
+	return GetTeamFilter(TEAM_INNOCENT, aliveOnly)
 end
 
 ---
--- Returns a list of all @{Players} of the Traitor team
--- @param boolean alive_only
+-- Returns a list of all @{Players} of the Traitor team.
+-- @param boolean aliveOnly
 -- @return table
 -- @realm server
 -- @see GetTeamFilter
-function GetTraitorFilter(alive_only)
-	return GetTeamFilter(TEAM_TRAITOR, alive_only)
+function GetTraitorFilter(aliveOnly)
+	return GetTeamFilter(TEAM_TRAITOR, aliveOnly)
 end
 
 ---
--- Returns a list of filtered @{Player}s by the @{ROLE}'s index
+-- Returns a list of filtered @{Player}s by the @{ROLE}'s index.
 -- @note If a BaseRole is given, this will return true for all its SubRoles.
 -- If you just want to filter for a specific SubRole, use @{GetSubRoleFilter} instead
 -- @param number subrole
--- @param boolean alive_only
+-- @param boolean aliveOnly
 -- @return table
 -- @realm server
 -- @see Player:IsRole
-function GetRoleFilter(subrole, alive_only)
+function GetRoleFilter(subrole, aliveOnly)
 	return GetPlayerFilter(function(p)
-		return p:IsRole(subrole) and (not alive_only or p:IsTerror())
+		return p:IsRole(subrole) and (not aliveOnly or p:IsTerror())
 	end)
 end
 
 ---
--- Returns a list of filtered @{Player}s by the @{ROLE}'s SubRole index
+-- Returns a list of filtered @{Player}s by the @{ROLE}'s SubRole index.
 -- @param number subrole
--- @param boolean alive_only
+-- @param boolean aliveOnly
 -- @return table
 -- @realm server
-function GetSubRoleFilter(subrole, alive_only)
+function GetSubRoleFilter(subrole, aliveOnly)
 	return GetPlayerFilter(function(p)
-		return p:GetSubRole() == subrole and (not alive_only or p:IsTerror())
+		return p:GetSubRole() == subrole and (not aliveOnly or p:IsTerror())
 	end)
 end
 
 ---
--- Returns a list of all @{Players} of the Detective @{ROLE}'s index
--- @param boolean alive_only
+-- Returns a list of all @{Players} of the Detective @{ROLE}'s index.
+-- @param boolean aliveOnly
 -- @return table
 -- @realm server
 -- @see GetRoleFilter
-function GetDetectiveFilter(alive_only)
-	return GetRoleFilter(ROLE_DETECTIVE, alive_only)
+function GetDetectiveFilter(aliveOnly)
+	return GetRoleFilter(ROLE_DETECTIVE, aliveOnly)
 end
 
 ---
--- Returns a list of all @{Players} of a specific @{ROLE}'s index that are able to chat
+-- Returns a list of all @{Players} of a specific @{ROLE}'s index that are able to chat.
 -- @param number subrole
--- @param boolean alive_only
+-- @param boolean aliveOnly
 -- @return table
 -- @realm server
-function GetRoleChatFilter(subrole, alive_only)
+function GetRoleChatFilter(subrole, aliveOnly)
 	if roles.GetByIndex(subrole).disabledTeamChat then
 		return {}
 	end
 
 	return GetPlayerFilter(function(p)
-		return p:IsRole(subrole) and not p:GetSubRoleData().disabledTeamChatRec and (not alive_only or p:IsTerror())
+		return p:IsRole(subrole)
+			and not p:GetSubRoleData().v
```

Comment:
i suppose you wanted to append the 'v' here?

Reply:
oh man, thanks! I accidentally replaced instead of extending it. Oof